runtime/compiler/build/toolcfg/aix-xlc/common.mk

```diff
@@ -92,7 +92,7 @@ CX_FLAGS_PROD+=$(CX_OPTFLAG)
 ifdef ENABLE_SIMD_LIB
     CX_DEFINES+=ENABLE_SPMD_SIMD
     CX_ARCH?=pwr7
-    CX_FLAGS+=-qaltivec -qtune=pwr7
+    CX_FLAGS+=-qaltivec -qtune=pwr7 -D__EXTABI__
```

Comment:
Do we need this flag everywhere (not just the JIT)?

Reply:
It is definitely needed where signal handling/longjmp/setjmp/context-related activities can happen.  or, where vr52-63 can potentially be touched (unlikely outside JIT-ed code though).

Reply:
The actual setjmp/longjmp for exception handling is in OMR. I agree it's extremely unlikely that any generic C code in the VM will need to use VSR at all, let alone the top-end ones, but if it's possible, then we'll need the define. The JCL code will also need to have it.